hack/common

```diff
@@ -135,8 +135,8 @@ function get_private_repo_dir() {
             done
             touch repos/redhat.repo
             chmod 0444 repos/redhat.repo
-            sed -i -e 's,^sslclientkey.*$,sslclientkey = /etc/yum.repos.d/ops-mirror.pem,' \
-                   -e 's,^sslclientcert.*$,sslclientcert = /etc/yum.repos.d/ops-mirror.pem,' repos/*.repo
+            gnu_sed -i -e 's,^sslclientkey.*$,sslclientkey = /etc/yum.repos.d/ops-mirror.pem,' \
```

Comment:
this command isn't on f31 -- can we move the command into a variable that we can update to be `gnu_sed` for `darwin` hosts?